lmdb/liblmdb/mdb.c

```diff
@@ -39,40 +39,40 @@
 #define _FILE_OFFSET_BITS	64
 #endif
 #ifdef _WIN32
-#include <malloc.h>
-#include <windows.h>
-#include <wchar.h>				/* get wcscpy() */
+	#include <malloc.h>
```

Comment:
There's a bunch of whitespace changes in this file that could go away, leaving just the actual functional change.